src/app/modules/shared_modules/keycard_popup/controller.nim

```diff
@@ -30,16 +30,21 @@ type
     keychainConnectionIds: seq[UUID]
     connectionKeycardResponse: UUID
     tmpKeycardContainsMetadata: bool
+    tmpCardMetadata: CardMetadata
```

Comment:
Wondering what is temporary to this data? I see that we set it but when is it discarded?

Reply:
@stefandunca it's set and discarded when the flow is done, cause it's part of the shared module which is created in the moment when we need to run some flow and destroyed after the flow is done.